src/vs/workbench/contrib/positronNotebook/common/editor/position.ts

```diff
@@ -0,0 +1,50 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Some helpers that wrap the existing position/range classes with a `cellIndex` as well